### ISSUE TITLE
Trigger informing editor

### ISSUE THREAD

**Issue by drjakob** [roschaefer/story.board#518]
As editor,
I want to be informend if some Triggers get activated
in order to explain difficult situations.

**Comment by roschaefer** [comment]
Is it enough to have green lights like in #374 for triggers also?

**Comment by drjakob** [comment]
No, the idea is to alert actively in urgent trigger situations to check if all sensor information is correct.(e.g. Animal is ill).

**Comment by roschaefer** [comment]
1. What is an *urgent* trigger?
2. What kind of notification?

**Comment by drjakob** [comment]
Priority=urgent
Notification=Push via email
Important: This is just an idea, not planned to be developped yet.

**Comment by roschaefer** [comment]
ah yeah, I was wrong, the priorities are on the trigger not the text components
